src/tests/test_analysis_intellij.py

```diff
@@ -8,25 +8,22 @@
 
 
 @pytest.mark.parametrize("app_name", json.load(open(DATA_DIR + "analysis.json")))
-def test_analysis_intellij(setup_intellij, app_name, analysis_data):
+@pytest.mark.parametrize("ide", ["intellij"])
```

Comment:
Why do we need to parametrize this for IDE? This test will only run for intelliJ right ?